.gitignore

```diff
@@ -28,3 +28,5 @@ test_file_text.txt
 # Untracked artifacts from the conda script
 repo2docker/buildpacks/conda/environment.py-3.5.yml
 repo2docker/buildpacks/conda/environment.py-3.6.yml
+
+\.vscode/
```

Comment:
For my (never ending) git education: what does the `\` at the start of the line do? To ignore a hidden directory with this name I would have put `.vscode/`.

Reply:
I though it would escape the `.` as a special character, but it works with `.vscode/` as well. Was added by the git client and I did not question it.